packages/cardano-services/src/ChainHistory/openApi.json

```diff
@@ -204,7 +204,13 @@
         "type": "object",
         "properties": {
           "blob": { "anyOf": [{ "$ref": "#/components/schemas/Map" }, { "$ref": "#/components/schemas/Undefined" }] },
-          "scripts": {
+          "nativeScripts": {
```

Comment:
Is this right? I think it's still just `scripts` array

Reply:
Good catch, I forgot to update it

Reply:
So it seems openapi doesn't validate the responses, only requests? I'm wondering if we can configure it to validate responses in test env

Reply:
I think it does, it just happens that those fields are optional and right now we are no retrieving them in any test it seems

Reply:
addressed here: https://github.com/input-output-hk/cardano-js-sdk/pull/388/commits/de5acdc64e9567ec43e6142e650a2607ba6a428c